pkg/common/connectionmanager/search.go

```diff
@@ -187,12 +191,18 @@ func (cm *ConnectionManager) WhichVCandDCByNodeID(ctx context.Context, nodeID st
 					continue
 				}
 
+				hostName := oVM.Guest.HostName
+				if searchBy == FindVMByIP {
+					klog.V(2).Info("WhichVCandDCByNodeID by IP. Overriding VMName from=%s to to=%s", oVM.Guest.HostName, myNodeID)
```

Comment:
this needs to be `.Infof` or it will not pass lint

Reply:
Good catch!